addons/web/static/src/views/fields/many2many_tags/many2many_tags_field.js

```diff
@@ -53,6 +53,9 @@ export class Many2ManyTagsField extends Component {
         });
 
         this.update = (recordlist) => {
+            if (!recordlist) {
```

Comment:
Commit message says:

> As the update method for m2m_tags expects a record or record list
this caused a callback.

The caused a "callback" ? Isn't it traceback ?

Reply:
Correct. Corrected!